.github/workflows/build-and-release-dev.yml

```diff
@@ -0,0 +1,151 @@
+name: Build a dev version for all packages and release them
+
+on:
+  pull_request_review:
+    types: [submitted]
+
+jobs:
+  build-and-release-dev:
+    timeout-minutes: 20
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ssh-key: ${{secrets.DEPLOY_KEY}}
+      - uses: actions/setup-python@v4
+        with:
+          python-version: 3.9
+      - uses: actions/setup-node@v4
+        with:
+          node-version: '20'
+
+      - name: Setup Dev Version
+        id: version-setup
+        run: |
+          python tools/release/setup_version.py ALL dev >> $GITHUB_OUTPUT
```

Comment:
You don't bump the version here? Is it on purpose?

Reply:
Just while testing, to no generate too many versions